$OpenBSD$

Fix warning: comparison between signed and unsigned,
   XXX necessary ? should update Ecore_X_Randr_Mode_Info ?

--- src/bin/e_randr.c.orig	Fri Nov 11 21:34:46 2011
+++ src/bin/e_randr.c	Mon Nov 14 10:15:19 2011
@@ -2039,7 +2039,7 @@ _e_randr_crtc_mode_intersects_crtcs(E_Randr_Crtc_Info 
             ((tmp->geometry.w <= 0) || (tmp->geometry.h <= 0)))
           continue;
         if (E_INTERSECTS(crtc_info->geometry.x, crtc_info->geometry.y,
-                         mode->width, mode->height, tmp->geometry.x,
+                         (int)mode->width, (int)mode->height, tmp->geometry.x,
                          tmp->geometry.y, tmp->geometry.w, tmp->geometry.h)
             && ((crtc_info->geometry.x != tmp->geometry.x) &&
                 (crtc_info->geometry.y != tmp->geometry.y)))
